rfd/0027-account-lifecycle.md

```diff
@@ -0,0 +1,195 @@
+---
+authors: Sasha Klizhentas (sasha@goteleport.com)
+state: draft
+---
+
+# RFD 27 - Account Lifecycle: Recovery and Cancellation
+
+## What
+
+Introduces cloud account recovery and cancellation flow.
+
+## Why
+
+Having a secure self-management account recovery and cancelation
+flow improves user experience and reduces support load on teams.
+
+## Details
+
+The recovery flow focuses on local accounts.
+SSO account recovery is out of scope, however UI will provide some hints.
+
+Each local user will receive 3 account recovery tokens on signup.
+
+Each token is a crypto-randomly generated 32 byte string that could be used only once.
+Once signed up, users will be presented with the recovery tokens printed on the
+screen with a message:
+
+```
+Please save these account recovery tokens in a safe offline place.
+You can use each token once if you loose your second factor.
+
+* recover-example-teleport-sh-token-1
```

Comment:
nit: use the actual domain name `recover-example.teleport.sh-token-1`

Reply:
done